utils/stringcomparator/stringcomparator.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package stringcomparator
+
+type StringComparator struct {
+	a string
+	b string
+}
+
+func New(a, b string) *StringComparator {
+	return &StringComparator{a: a, b: b}
+}
+
+// levenshteinDistance
```

Comment:
Method comments need to start with the name of the method and be a sentence.

Reply:
Thanks Ian will update this :)